src/coreComponents/physicsSolvers/multiphysics/benchmarks/Proppant-transport-validation-case.xml

```diff
@@ -0,0 +1,328 @@
+<?xml version="1.0" ?>
```

Comment:
Should be remove the example here: https://github.com/GEOSX/GEOSX/blob/develop/examples/proppantTransport/Proppant-transport-validation-case.xml
and just keep the xml you are adding in this PR?